src/test/kotlin/io/github/rybalkinsd/kohttp/dsl/UploadDslKtTest.kt

```diff
@@ -9,7 +10,21 @@ class UploadDslKtTest {
 
     @Test
     fun `small file upload`() {
-        val r  = upload {
+        val r = upload {
+
+            header {
```

Comment:
there is no assertions for `header` and `param` in this test. Also there is a test `test file upload with headers and parameters`. So it's probably not necessary to include them in this test